### PR TITLE
Switch to Stamen Toner basemap

### LINKED ISSUE #515: Swap in Stamen Toner basemap

### PR DESCRIPTION
**Pull request by jwalgran**
Parks likes the general look of Toner. Swapping in to see how it looks in context.

Fixes #515

### CODE REVIEW

File: src/nyc_trees/js/src/map.js [hunk 1]
```diff
@@ -72,11 +72,17 @@ function fitBounds(map, bounds) {
 }
 
 function initBaseMap(map) {
-    var osmUrl = 'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
-        osmAttrib = 'Map data © <a href="http://openstreetmap.org">OpenStreetMap</a> contributors',
-        osm = new L.TileLayer(osmUrl, {maxZoom: zoom.MAX, attribution: osmAttrib});
+    var url = 'http://{s}.tile.stamen.com/tonerg/{z}/{x}/{y}.png',
+        options = {
+            subdomains: 'abcd',
```
Comment (steventlamb) [inline src/nyc_trees/js/src/map.js]: for my education, why the subdomains?

Comment (jwalgran) [inline src/nyc_trees/js/src/map.js]: They get plugged in as values for ``{s}``. Browsers will often only load 2 resources at a time from a given domain. Using multiple subdomains allows for more tile requests in parallel.

**Comment by steventlamb** [comment]
I pulled this down, reprovisioned, restarted the debugserver and hit this url:
http://localhost:8000/progress/

I'm getting 400s on tile requests.
screenshot from 2015-02-13 15 00 13

**Comment by jwalgran** [comment]
Sorry to waste your time. ``tonerg`` --> ``toner``.

**Comment by steventlamb** [comment]
+1 works

**Pull request merged**